CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 # UNRELEASED
 
+* IntuneDeviceConfigurationFirmwareInterfacePolicyWindows10
+  * Initial release.
+* O365SearchAndIntelligenceConfigurations
+  * Initial release.
+* DRG
+  * Fixed layout and display issues in module file
+
+# 1.23.405.1
```

Comment:
Please remove this header (duplicate) and incorporate the changes into the lines below